src/lib/runtime.ts

```diff
@@ -73,7 +73,7 @@ export class BaseAPI {
       method: context.method,
       headers,
       body: context.body,
-      agent: this.configuration.agent,
+      dispatcher: this.configuration.agent,
```

Comment:
I think we should ensure `this.configuration.agent` is of type `Dispatcher` from `undici`, else it will throw an error if I recall correctly. Or would it work if we pass `https.Agent` here ?

Reply:
Updated the agent to type `Dispatcher` from `undici` as suggested in this commit https://github.com/auth0/node-auth0/pull/1002/commits/0d041f4bfe706566ec014ff518df7ed49206311e